### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by LixvYang** [vuepress-theme-hope/vuepress-theme-hope#3432] [Q&A] [answered]
### Checklist

- [X] I am using **LTS** version of Node.js.
- [X] I am using the **latest v2 version** of VuePress and VuePress official plugins.
- [X] I am using the **latest v2 version** of vuepress-theme-hope and all plugins in this repo.
- [X] I followed the docs and I double checked my configuration.
- [X] I have searched [FAQ](https://theme-hope.vuejs.press/faq/) and I did not find the answer.

### Package name

vuepress-theme-hope

### Which operating system are you using?

- [X] macOS
- [ ] Windows
- [ ] Linux

### Describe the bug

The file that generates the Github Action needs to be updated or the build will fail.

- name: install pnpm
        uses: pnpm/action-setup@v2
        with: pnpm/action-setup@v2
          version: 8
          run_install: |
            - recursive: true
              args: [--no-frozen-lockfile]

### Additional context

Auto Generate .github/workflows/deploy-docs.yaml ERROR!

**Comment by Mister-Hope** [comment]
Why `--no-frozen-lockfile` is needed? And why a normal repo should use recursive install?

**Comment by Mister-Hope** [comment, marked as answer]
Closed as no response for a month
